zio-flow/shared/src/main/scala/zio/flow/remote/RemoteCharSyntax.scala

```diff
@@ -0,0 +1,52 @@
+///*
+// * Copyright 2021-2022 John A. De Goes and the ZIO Contributors
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// */
+
+package zio.flow.remote
+
+import zio.flow._
+import zio.flow.Remote.Literal
+import zio.schema.{DynamicValue, StandardType}
+
+final class RemoteCharSyntax(val self: Remote[Char]) extends AnyVal {
+  def toUpperCase: Remote[Char] =
+    overLiteralMakePrimitive[Char](self, _.toUpper)
+
+  def toLowerCase: Remote[Char] =
+    overLiteralMakePrimitive[Char](self, _.toLower)
+
+  def toTitleCase: Remote[Char] =
+    overLiteralMakePrimitive[Char](self, _.toTitleCase)
+
+  private def overLiteralMakePrimitive[A](x: Remote[A], k: A => A): Remote[A] =
+    overLiteralPrimitive[A](x, (value, p) => DynamicValue.Primitive(k(value.asInstanceOf[A]), p))
+
+  private def overLiteralPrimitive[A](
```

Comment:
I don't think this is a good idea, we can't assume that a `Remote[Char]` is `Literal`.
For example if you have this code:
`(tuple: Remote[(Char, Int)]) => tuple._1.toUpperCase` then the `Remote[Char]` it's called on is a `Remote.TupleAccessor`, or it could be a remote function, etc.